great_expectations/core/expectation_diagnostics/supporting_types.py

```diff
@@ -179,6 +179,7 @@ class ExpectationDiagnosticCheckMessage(SerializableDictDot):
 class ExpectationDiagnosticMaturityMessages(SerializableDictDot):
     """A holder for ExpectationDiagnosticCheckMessages, grouping them by maturity level. Used within the ExpectationDiagnostic object."""
 
+    concept_only: List[ExpectationDiagnosticCheckMessage]
```

Comment:
I don't think we need this.

Yes, `concept_only` is a valid maturity level, but it's the default maturity level for Expectations that don't yet even pass the checks required for `experimental.` So there can't be any `ExpectationDiagnosticCheckMessages` associated with `concept_only`.